server/Configurations/googleAuth/passportGoogleConfig.js

```diff
@@ -0,0 +1,43 @@
+require("dotenv").config();
```

Comment:
Nice implementation 👍 
Consider using spaces in between the different action blocks happening in your code. This will make your file easier to read and follow.